sympy/matrices/tests/test_reductions.py

```diff
@@ -1,4 +1,4 @@
-from sympy import symbols, simplify, I
+from sympy import symbols, I
```

Comment:
Remove comma after symbols too

Reply:
I have already done that, thanks though.

Reply:
Also, error is on line 9 too as sugggested by flake8, remove the simplify import

Reply:
Have done that, please check.